src/components/molecules/LiveSchedule/LiveEvent.tsx

```diff
@@ -43,11 +39,26 @@ export const LiveEvent: FC<LiveEventProps> = ({ venue, event }) => {
           {event.description}
         </div>
         <div className="schedule-event-info-room">
-          <div onClick={enterLiveEvent}>
-            {event.room ?? "Enter"} {venue && `- ${venue.name}`}
-          </div>
+          {event.room && room ? (
+            <EnterRoomButton room={room as Room}>{buttonText}</EnterRoomButton>
```

Comment:
I don't feel that this is the best way to solve this. I think it is better than before, but can be improved. I'm opened to suggestions